core/src/test/java/org/apache/calcite/test/SqlHintsConverterTest.java

```diff
@@ -931,6 +937,12 @@ private static class HintCollector extends RelShuttleImpl {
           if (snapshot.getHints().size() > 0) {
             this.hintsCollect.add("Snapshot:" + snapshot.getHints());
           }
+        } else if (other instanceof LogicalTableFunctionScan) {
```

Comment:
Prefer to use the base class `TableFunctionScan`.

Reply:
Updated